test/upload.test.js

```diff
@@ -60,7 +60,7 @@ const file = new Open.NodeFileSystemFile('./test/fixtures/sample.csv')
 /**
  * Mock
  */
-const ckanAuthzMock = nock("http://localhost:5000")
+const ckanAuthzMock = nock("http://127.0.0.1")
   .persist()
   .post('/api/3/action/authz_authorize', ckanAuthzConfig.body)
```

Comment:
Below also commented about this. So basically I would like to assert the `body`, just to make sure the funciton pass the right body. As an example you can see [here](https://github.com/datopian/ckan-client-js/blob/master/test/action.test.js#L38)

Reply:
I left some comments above.